sdcm/cluster.py

```diff
@@ -3330,6 +3330,7 @@ def configure_scylla_monitoring(self, node, sct_metrics=True, alert_manager=True
                 templ_yaml = yaml.load(f, Loader=yaml.SafeLoader)  # to override avocado
                 self.log.debug("Configs %s" % templ_yaml)
             loader_targets_list = ["%s:9103" % n.ip_address for n in self.targets["loaders"].nodes]
+            gemini_loader_targets_list = ["%s:2112" % n.ip_address for n in self.targets["loaders"].nodes]
```

Comment:
run this only if Gemini is intended to run

Reply:
fixed